### DISCUSSION TITLE
Get the redirecturl when the user create an account

### DISCUSSION THREAD

**Discussion by MoathCodes** [saleor/saleor#11757] [Q&A] [answered]
Hi,
I would like to know if there is away to get redirectUrl when the user create an account in order to send it to the user through something else other than the email like an SMS message.
Thank you.

Update: now that I think about it I don't need the redirectUrl it's the token within the rediructUrl that is used to setPassword.

**Comment by peelar** [comment]
Hi @IronDragonDev 👋 

We have [`redirectUrl`](https://docs.saleor.io/docs/3.x/api-reference/inputs/account-register-input#code-style-fontweight-normal-accountregisterinputbredirecturlbcodestring-) field in the [`AccountRegisterInput`](https://docs.saleor.io/docs/3.x/api-reference/inputs/account-register-input#code-style-fontweight-normal-accountregisterinputbredirecturlbcodestring-) that is used in the [`accountRegister`](https://docs.saleor.io/docs/3.x/api-reference/mutations/account-register#code-style-fontweight-normal-accountregisterbinputbcodeaccountregisterinput--) mutation.

Does that solve your problem?

**Reply by MoathCodes** [reply]
I'm sorry I didn't do a good job at explaining what do I need,
What i meant is I wanted the account to be confirmed and activated without the need to get an email, something like the `ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL` but for SaleorCloud.

To put it in a simpler way, I want the confirmation process to be on my end not on saleor API.

**Comment by MoathCodes** [comment, marked as answer]
Solved the problem by creating an app that has USER_MANAGEMENT role, then used the app token to update the customer account, setting the isActive field to true.
